aten/src/ATen/core/Formatting.cpp

```diff
@@ -281,7 +281,8 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
       if (tensor_.qscheme() == c10::kPerTensorAffine) {
         stream << ", scale: " << tensor_.q_scale();
         stream << ", zero_point: " << tensor_.q_zero_point();
-      } else if (tensor_.qscheme() == c10::kPerChannelAffine) {
+      } else if (tensor_.qscheme() == c10::kPerChannelAffine ||
+          tensor_.qscheme() == c10::kPerRowFloatQParams) {
```

Comment:
Can we call this PerChannelFloatQParams as this generalized per-channel quant?

Reply:
Yes, I will be renaming these shortly and pushing an update.